I'm echoing to stdout
I'm echoing to stderr
My JobID is 55534678
I have 4 CPUs on node r108u25n01
Importing...
/home/bsw38/.conda/envs/zinc_testing/lib/python3.13/site-packages/outdated/__init__.py:36: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  from pkg_resources import parse_version
Imports complete
Hello, world!
Traceback (most recent call last):
  File "/vast/palmer/home.mccleary/bsw38/GraphML/ZINC/main.py", line 248, in <module>
    main(eigMod=eigMod, num_eigenvectors=num_eigenvectors)
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/vast/palmer/home.mccleary/bsw38/GraphML/ZINC/main.py", line 143, in main
    train_ds = [toGraphObject(graph) for graph in dataset_hf['train']]
                ~~~~~~~~~~~~~^^^^^^^
  File "/vast/palmer/home.mccleary/bsw38/GraphML/ZINC/main.py", line 141, in toGraphObject
    encoded = encoder(graph['node_feat'])
  File "/home/bsw38/.conda/envs/zinc_testing/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/bsw38/.conda/envs/zinc_testing/lib/python3.13/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/bsw38/.conda/envs/zinc_testing/lib/python3.13/site-packages/ogb/graphproppred/mol_encoder.py", line 21, in forward
    for i in range(x.shape[1]):
                   ^^^^^^^
AttributeError: 'list' object has no attribute 'shape'
